frontend/src/components/Selectors/FilterMethodSelector.tsx

```diff
@@ -18,7 +19,14 @@ let menuItems = chain(options).map(
 export let FilterMethodSelector: React.StatelessComponent<Props> = ({ className, onChange, value }) =>
   <DropDownMenu
     className={className ? className : undefined}
-    onChange={(_event, _index, value) => onChange(value)}
+    onChange={(_event, _index, value) => {
+      ReactGA.event({
+        category: 'Filter',
+        action: 'Selected a filter type',
```

Comment:
filter method maybe?

Reply:
"Filter Type" was the language @viviandien used